app/src/main/res/values/themes.xml

```diff
@@ -13,4 +13,9 @@
         <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>
         <!-- Customize your theme here. -->
     </style>
+
+    <!--Theme for SplashScreen-->
+    <style name="SplashTheme" parent="Theme.HackTziBlog">
```

Comment:
Awesome choice for create a new style. Just to confirm that it looks good in other devices. Could you please test it in device that support light/dark mode? :

Reply:
Ok, already I confirm that it looks good in both  themes, I will send a PR